frontend/src/main/scala/bloop/cli/Commands.scala

```diff
@@ -45,11 +45,11 @@ object Commands {
   ) extends RawCommand
 
   /** List of commands accepting a project as argument, used for autopomcletion */
-  val projectBound = CommandsMessages.messages
-    .collect {
-      case (name, CommandMessages(args, _)) if args.exists(_.name == "project") => name
-    }
-    .mkString(" ")
+  val projectBound: String = {
```

Comment:
Did you use a different scalafmt configuration to format the sources? In my previous commit, I used the `scalafmt` command within sbt.

Reply:
In this case, scalafmt formats it differently if the collect is only in one line. I happen to know these tricks and that's why I reformatted so that it reads better :smile:

Reply:
I see. I was under the impression that scalafmt's formatters were deterministic.

Reply:
Oh they are, but in this case I moved the collect to a new line and that wraps it’s body to the same line if it fits.